### PR TITLE
Sending AD information when calling getssodata

### PR DESCRIPTION
**Pull request by luisrudge**
We're not sending the `AD` flag to auth0.js when we call getSSOData. This is fine for embedded since we ignore this flag. But for hosted pages, which use the /ssodata endpoint, this flag is required in order to return information about AD connections.

### CODE REVIEW

File: src/core/web_api/p2_api.js [hunk 1]
```diff
@@ -123,17 +123,8 @@ class Auth0APIClient {
     this.getUserInfo(token, callback);
   }
 
-  getSSOData(cb) {
-    if (this.isUniversalLogin) {
```
Comment (cocojoe) [inline src/core/web_api/p2_api.js]: This logic is no longer required?

**Comment by cocojoe** [comment]
@luisrudge please rebase and add some details to the description? 
What was the problem? Why should it be fixed?

File: src/core/web_api/p2_api.js (hunk 1 above)
Comment (luisrudge) [reply src/core/web_api/p2_api.js]: nope, auth0.js has all the logic to handle universal login

**Comment by luisrudge** [comment]
@cocojoe done

**Pull request merged**